main.py

```diff
@@ -523,6 +556,25 @@ def delete_link_metadata(self, link_id, key):
         self.notify_metadata_changes(link, 'removed')
         return jsonify("Operation successful"), 200
 
+    @listen_to("kytos/mef_eline.link_available_tags")
```

Comment:
```suggestion
    @listen_to("kytos/*.link_available_tags")
```
If some other NApp uses tags.

Reply:
Good idea @ajoaoff, let's use this wildcard.